lib/api/api1.js

```diff
@@ -939,6 +967,7 @@ module.exports = {
    * @param {object} request the request input object
    * @param {object} metadata cache
    * @param {object} error error collector
+   * @param {boolean} [strict] if true, raise error for unknown criteria
```

Comment:
What's the "strict" modifier mean here?  Parameter must be an actual `Boolean` rather than simply something truthy/falsy?  (I haven't seen this in JSDoc before)

Reply:
`strict` is the parameter name. If you enclose it in brackets, it means that it is optional.

https://jsdoc.app/tags-param.html#optional-parameters-and-default-values